# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../bsd-games/dm-noutmpx.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- bsd-games-2.16/dm/Makefrag.orig	2005-08-02 19:14:38.000000000 -0700
+++ bsd-games-2.16/dm/Makefrag	2005-08-02 19:14:54.000000000 -0700
@@ -27,7 +27,7 @@
 # SUCH DAMAGE.
 
 # Add -DLOG if you want logging
-dm_DEFS := $(GETLOADAVG_DEFS) -D_GNU_SOURCE -DSUPPORT_UTMPX -DSUPPORT_UTMP # -DLOG
+dm_DEFS := $(GETLOADAVG_DEFS) -D_GNU_SOURCE -DSUPPORT_UTMP # -DLOG
 dm_DIRS := $(GAMESDIR) $(MAN8DIR) $(MAN5DIR)
 
 dm_all:	dm/dm dm/dm.8 dm/dm.conf.5
